java/src/apps/gui3/tabbedpreferences/EditConnectionPreferencesDialog.java

```diff
@@ -40,8 +40,12 @@ public boolean isMultipleInstances() {
     public static boolean showDialog() {
         EditConnectionPreferencesDialog dialog = new EditConnectionPreferencesDialog();
         SwingUtilities.updateComponentTreeUI(dialog);  // hack because sometimes this was created before L&F was set?
-        
         dialog.pack();
+        dialog.setBounds(
+                (int) ( jmri.util.JmriJFrame.getScreenDimensions().get(0).getBounds().getSize().getWidth() * 0.10 ),
+                (int) ( jmri.util.JmriJFrame.getScreenDimensions().get(0).getBounds().getSize().getHeight() * 0.10 ),
+                (int) ( jmri.util.JmriJFrame.getScreenDimensions().get(0).getBounds().getSize().getWidth() * 0.80 ),
+                (int) (jmri.util.JmriJFrame.getScreenDimensions().get(0).getBounds().getSize().getHeight()* 0.80 ));
```

Comment:
A question:
Do we know that the window will be shown on the screen 0 ?

If the window is shown on another screen and that screen is smaller, we have a problem. But I don't know if the window might show on another screen or not.

Reply:
@danielb987 
The bounds sets the origin 10% in and 80% of the first screen.
Depending on how people configure multiple screens it many be displayed incompletely on multiple screens and completely on one or more screens.
Under Wayland it is placed where the composter thinks it fits best, but will put it where it's told if it agrees with our choice. Welcome to 1984.

Reply:
> Under Wayland it is placed where the composter thinks it fits best, but will put it where it's told if it agrees with our choice.

Is this documented anywhere? If I remember correctly, @mstevetodd or you mentioned something about the design decisions the Wayland developers have done about this. It would be interesting to read about why the design is this way.

Reply:
@danielb987 
Daniel, Daniel Stone is/was one of the movers and shakers. 
[https://www.mail-archive.com/wayland-devel@lists.freedesktop.org/msg41613.html](url)
He has a number of youtube presentations on wayland.
Depending on your view this is either a rant or measured article,
[https://dudemanguy.github.io/blog/posts/2022-06-10-wayland-xorg/wayland-xorg.html](url)
The is a wikipedia article on wayland that is reasonable unbiased.
[https://en.wikipedia.org/wiki/Wayland_(protocol)](url)
A quick and simple article.
[https://mir-server.io/docs/window-positions-under-wayland](url)